projects/libtiff/tiff_read_rgba-afl++.toml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+[sydr]
+args = "-l debug -j2"
+target = "/tiff_read_rgba_sydr @@"
+jobs = 2
+
+[[aflplusplus]]
```

Comment:
change `[[]]` to `[]`, we don't have multiple afl tables here.

Reply:
done